ci-operator/config/openshift/oc/openshift-oc-release-4.10.yaml

```diff
@@ -23,6 +23,11 @@ binary_build_commands: make build --warn-undefined-variables
 build_root:
   from_repository: true
 images:
+- dockerfile_literal: |
```

Comment:
the great news are that this image has been created successfully. take a look at: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/25752/rehearse-25752-pull-ci-openshift-oc-release-4.10-images/1488587798242922496/artifacts/build-logs/cli-with-jq.log

the less good news (not that bad though) is that it did failed to access some sources that are configured (all those ``localdev-*-rpms``). So I'll suggest you continue with your work, but do drop a message on slack channel ``#forum-testplatform`` so that they can take a look and maybe remove those sources, or correct them

Reply:
We don't own content of any images